src/System.Drawing.Primitives/System.Drawing.Primitives.targets

```diff
@@ -0,0 +1,27 @@
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="_BuildSystemDrawingPrimitivesFacade">
+   <MSBuild Projects="..\Mono.Android\Mono.Android.csproj"
+             Targets="GetTargetPath"
```

Comment:
Indentation conventions for .csproj/.targets files:

1. Spaces, not tabs. (Not that you have tabs...)
2. *2*-space "tab stops".
3. If attributes are on a new line, then they should be indented an "extra" tab stop, so that they're indented *more* than child content.
4. `<Output/>` should be on a single line.

Applying the above:

```xml
<MSBuild Projects="..\Mono.Android\Mono.Android.csproj"
    Targets="GetTargetPath"
    Properties="Configuration=$(Configuration)">
  <Output TaskParameter="TargetOutputs" ItemName="MonoAndroidAssembly" />
</MSBuild>
```